src/fal/packages/dependency_analysis.py

```diff
@@ -118,6 +118,7 @@ def _get_fal_package_name() -> Tuple[str, Optional[str]]:
 def get_default_requirements() -> Iterator[Tuple[str, Optional[str]]]:
     yield _get_fal_package_name()
     yield from _get_dbt_packages()
+    yield "isolate", importlib_metadata.version("isolate")
```

Comment:
We probably should copy this function to dbt-fal code and change it as needed there, we don't use isolate in fal, right?